relm4/src/binding/bindings.rs

```diff
@@ -41,6 +41,7 @@ macro_rules! binding {
         }
 
         #[allow(missing_docs)]
+        #[allow(clippy::must_use_candidate)]
```

Comment:
This is a regression in `glib-macros` and I've just sent a PR to fix this. Ironically, the have the same problem: they recently changed `quote` to `quote_spanned` (with nonsensical spans) and now clippy is throwing a warning that wasn't there before.

Which means that this should not be fixed from our side (even if it breaks the CI temporarily).

Reply:
`quote_spanned` does make sense there, as problems regarding the function are now properly attributed to the `#[property]` helper attribute instead of the `Properties` identifier in the `#[derive]` attribute. Using `quote_spanned` is generally good practice in more complex macros. The problem with `quote_spanned` in Relm4 are the double underscore identifiers created in the `quote_spanned` invocation.

Reply:
Yeah, the actual problem in Relm4 is that I've started with `quote` which resulted in poor error reporting and then thought that `quote_spanned` would be always better, which was sometimes true and sometimes not. And then I just kept it as it was.

It's not easy to do generate properly spanned code with quote in general though, because if I think about it, you'd probably need to split some `quote` invocations into several `quote` and `quote_spanned` invocations and merge their output into one stream to get it right in some places...

Reply:
Anyway, `#[allow(clippy::must_use_candidate)]` still needs to be removed before merging, because it's not a problem caused by Relm4 and was fixed in glib-macros (just not published on crates.io yet).

Reply:
```suggestion
```

I'll remove it. Do you have any further concerns regarding this PR?